### DISCUSSION TITLE
(i didnt make it but figured it out) MODCHARTS ON MAC BOYSS

### DISCUSSION THREAD

**Discussion by backhop** [KadeArchive/Kade-Engine#2759] [Show and tell]
ok so basically you just like o h my god you ok so oh jesus oh lord o h my yo u have to reinstall hxvm luajit and linc luajit i think aned th en you like uh u h h m m like edit project.xml anmd and it WORKS!!!! !!!!! !

https://user-images.githubusercontent.com/27576274/145678911-038f05fd-5ccb-495c-ac43-31bc9efed514.mov

heres my bad thing to show off

**Comment by isakube** [comment]
this guide was written by a vs bob dev 100%

**Comment by backhop** [comment]
> this guide was written by a vs bob dev 100%

somehow no
